code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -1,124 +1,148 @@
+		/obj/item/chameleon, // 7TC
+		/obj/item/encryptionkey/syndicate) // 2TC
 
-	var/static/list/thief = list( // 39TC
+	var/static/list/agent13 = list( // 37TC
+		/obj/item/clothing/under/chameleon, // 1TC
+		/obj/item/card/id/syndicate, // 2TC
+		/obj/item/gun/projectile/automatic/pistol, // 4TC
+		/obj/item/suppressor, // 1TC
+		/obj/item/ammo_box/magazine/m10mm,  // 1TC
+		/obj/item/ammo_box/magazine/m10mm/hp, // 2TC
+		/obj/item/twohanded/garrote, // 6TC
+		/obj/item/door_remote/omni/access_tuner, // 6TC
+		/obj/item/clothing/glasses/chameleon/thermal, // 6TC
+		/obj/item/implanter/freedom, // 6TC
+		/obj/item/coin/gold, // 0TC
+		/obj/item/encryptionkey/syndicate) // 2TC
+
+	var/static/list/thief = list( // 37TC
 		/obj/item/gun/energy/kinetic_accelerator/crossbow, // 12TC
 		/obj/item/chameleon, // 7TC
 		/obj/item/clothing/glasses/chameleon/thermal, // 6TC
 		/obj/item/clothing/gloves/color/black/thief, // 6TC
 		/obj/item/card/id/syndicate, // 2TC
-		/obj/item/clothing/shoes/chameleon/noslip, // 2TC
 		/obj/item/storage/backpack/satchel_flat, // 2TC
 		/obj/item/encryptionkey/syndicate) // 2TC
 
-	var/static/list/bond = list( // 33TC + three 0TC
+	var/static/list/bond = list( // 32TC
 		/obj/item/gun/projectile/automatic/pistol, // 4TC
 		/obj/item/suppressor, // 1TC
-		/obj/item/ammo_box/magazine/m10mm/hp,  // 3TC
+		/obj/item/ammo_box/magazine/m10mm/ap, // 2TC
 		/obj/item/ammo_box/magazine/m10mm/ap, // 2TC
 		/obj/item/clothing/under/suit/really_black, // 0TC
 		/obj/item/card/id/syndicate, // 2TC
 		/obj/item/clothing/suit/storage/lawyer/blackjacket/armored, // 0TC
+		/obj/item/storage/box/syndie_kit/emp, // 2TC
+		/obj/item/clothing/glasses/hud/security/chameleon, // 2TC
 		/obj/item/encryptionkey/syndicate, // 2TC
 		/obj/item/reagent_containers/food/drinks/drinkingglass/alliescocktail,	// 0TC
-		/obj/item/dnascrambler, // 4TC
-		/obj/item/storage/box/syndie_kit/emp, // 2TC
+		/obj/item/dnascrambler, // 2TC
 		/obj/item/CQC_manual) // 13TC
 
-	var/static/list/sabotage = list( // 41TC + two 0TC
-		/obj/item/grenade/plastic/c4, // 1TC
-		/obj/item/grenade/plastic/c4, // 1TC
-		/obj/item/camera_bug, // 1TC
-		/obj/item/powersink, // 10TC
-		/obj/item/cartridge/syndicate, // 6TC
+	var/static/list/infiltrator = list( // 34TC + RCD & Mesons Autoimplanter
+		/obj/item/storage/box/syndie_kit/teleporter, // 8TC
+		/obj/item/clothing/gloves/color/black/krav_maga, // 10TC
+		/obj/item/clothing/glasses/chameleon/thermal, // 6TC
+		/obj/item/pinpointer/advpinpointer, // 4TC
 		/obj/item/rcd/preloaded, // 0TC
-		/obj/item/card/emag, // 6TC
-		/obj/item/clothing/gloves/color/yellow, // 0TC
-		/obj/item/grenade/syndieminibomb, // 6TC
-		/obj/item/grenade/clusterbuster/n2o, // 4TC
 		/obj/item/storage/box/syndie_kit/space, // 4TC
+		/obj/item/autosurgeon/organ/syndicate/meson_eyes, // 0TC
 		/obj/item/encryptionkey/syndicate) // 2TC
 
-	var/static/list/payday = list( // 36TC + four 0TC
+	var/static/list/payday = list( // 35TC
```

Comment:
```suggestion
	var/static/list/payday = list( // 36TC
```
This is still 36 TC if you add up the stated values

Reply:
It's 35TC, I just forgot to update the jammer to be 4TC in the comments so I corrected that.